components/sendloop/sources/new-email-open/new-email-open.mjs

```diff
@@ -0,0 +1,43 @@
+import common from "../common/base.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "sendloop-new-email-open",
+  name: "New Email Open",
```

Comment:
```suggestion
  name: "New Email Opened",
```
Seems like this would be more clear (file/folder names and component key would be changed as well) - unless this is a very specific business name? ("email open" as a noun/entity)